cartridge/issues.lua

```diff
@@ -355,6 +361,27 @@ local function list_on_instance(opts)
         end
     end
 
+    -- add custom issues from each role
+    local registry = require('cartridge.service-registry')
+    for role_name, role in pairs(registry.list()) do
+        if role.get_issues ~= nil then
+            for _, issue in ipairs(role.get_issues()) do
```

Comment:
Check it's a function and pcall it. See a similar code in `roles.lua`:

https://github.com/tarantool/cartridge/blob/master/cartridge/roles.lua#L342-L351